### PR TITLE
:bug: Fix: follow-up of #1852

### PR DESCRIPTION
**Pull request by Xunzhuo**
**What type of PR is this?**

**What this PR does / why we need it**:

+ Add missing index
+ Set EnvoyProxy Service Type to ClusterIP, since LoadBalancer is not supported in default submariner env.
+ Upgrade RG to v1beta1.

### CODE REVIEW

**Review by zirain — APPROVED** [review]

File: examples/kubernetes/multicluster-service.yaml [hunk 1]
```diff
@@ -1,9 +1,29 @@
+apiVersion: config.gateway.envoyproxy.io/v1alpha1
+kind: EnvoyProxy
+metadata:
+  name: custom-proxy-config
+  namespace: envoy-gateway-system
+spec:
+  logging:
+    level:
+      default: warn
```
Comment (zhaohuabing) [inline examples/kubernetes/multicluster-service.yaml:9]: Is the logging level setting necessary?

Comment (Xunzhuo) [reply examples/kubernetes/multicluster-service.yaml:9]: Not necessary but does not impact anything, `default/warn` is the default value of logging.

Comment (zhaohuabing) [reply examples/kubernetes/multicluster-service.yaml:9]: Can we remove it since the default logging level is  already `warn`.

Comment (Xunzhuo) [reply examples/kubernetes/multicluster-service.yaml:9]: Done

**Review by zhaohuabing — APPROVED** [review]
LGTM. Thanks for cleaning up!

**Review by zirain — APPROVED** [review]

**Pull request merged**